README.md

```diff
@@ -1,19 +1,55 @@
 # ui
 
+# Get started
+
+This project is using [pnpm](https://pnpm.io/) as a package manager. To setup the monorepo run the following:
+
+```
+
+npm install -g pnpm
+cd ui
+pnpm install
+
+```
+
 This project is using [pnpm](https://pnpm.io/) as a package manager.
 
 ### Build
 
 To build all apps and packages, run the following command:
 
 ```
-pnpm run build
+
+pnpm  build
+
 ```
 
 ### Develop
 
 To develop all apps and packages, run the following command:
 
 ```
-pnpm run dev
+
+pnpm  dev
+
+```
+
+### Testing
+
+To run all test in the monorepo:
+
+```
+
+pnpm test
+
+```
+
+Turborepo will ran all test in your application
+
+To run jest on individual packages or apps
+
+```
+cd apps/docs or cd/web
+pnpm test
+
```

Comment:
I think we don't need this... just like we don't need it under **Build** or **Develop**